addons/udes_stock/models/procurement.py

```diff
@@ -8,3 +8,42 @@ class ProcurementGroup(models.Model):
 
     # Allow groups to be created
     MSM_CREATE = True
+
+    def _get_orderpoint_domain(self, company_id=False):
+        """ Override order point domain to filter by active_ids
+        """
+        domain = super()._get_orderpoint_domain(company_id=company_id)
+        orderpoints = self.env.context.get('active_ids', None)
+        if orderpoints:
+            domain += [('id', 'in', orderpoints)]
+        return domain
+
+        
+class StockWarehouseOrderPoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.model
+    def check_order_points(self, use_new_cursor=False, company_id=False):
+        """  Copy of run_scheduler to only check order points """
```

Comment:
A little more explanation would be helpful here, for example:
- why would we use a new cursor?
- where is the run_scheduler that we are copying?

Reply:
It's copied  from odoo's stock procurement.py